### ISSUE TITLE
Component - Info blocks

### ISSUE THREAD

**Issue by twblack88** [makerdao/community-portal#25]
This component should be thought of as "marketing blocks" AKA header image or inline image or containers. They're not related to particular user tasks as such but help give users reasons to believe in Maker and comm-dev. Plus they give designers opportunities to style the images later in the future.

[Extra detail](https://www.notion.so/Components-e4455d1797f74c48aae3019692a39413#0671bc4abfdd47618760d31c69f1a3f4)
[Frame from home page](https://www.figma.com/file/mVblaaFzkqracWycXl9czh/Maker-comm-dev?node-id=294%3A295)

**Comment by as-dr** [comment]
@MaximumCrash If the structure of the info blocks is essentially a block of written content alongside a visual image, you could:

1. Use `flex-wrap` to let them wrap to new lines as screen real estate reduces. [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-wrap)
1. Use `order` to ensure the images and written content are ordered in the desired manner (the accessibility issues around this are mitigated since we would be reordering an image and a piece of written copy, rather than two pieces of written copy) [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/order)

**Comment by MaximumCrash** [comment]
TLDR: Executing the best design for an MDX file that's written once will lead to inconsistencies in good design for these components on mobile.

**Comment by MaximumCrash** [comment]
Missing mobile styles, need to wait on how we'll be handling sidebar to get into breakpoints. However this issue is completed by #48
